lib/vmdb/loggers/formatters/container.rb

```diff
@@ -0,0 +1,47 @@
+class ContainerFormatter < Logger::Formatter
```

Comment:
Yeah, naming is hard.  I don't think this format is specific to fluent though...